releasenotes/notes/report-rpc.service-tag-e6363657f2ccbd5d.yaml

```diff
@@ -0,0 +1,29 @@
+---
+#instructions:
```

Comment:
Let's delete all the commented code here

Reply:
Done